Makefile.toml

```diff
@@ -172,4 +174,5 @@ script = ["""
   rm -f "${DART_WORKSPACE}"/lib/src/common/reranker/assets.dart
   rm -f "${DART_WORKSPACE}"/lib/src/web/reranker/assets.dart
   rm -f out/assets_metadata.json
+  rm -rf out/wasm_bindings
```

Comment:
can we add a cleanup for all copied files in the `bindings/dart/example/assets/*/` subdirectories, except for the `.gitkeep` file?

Reply:
Yes, I have added it to the makefile task.